analyses/cms-open-data-ttbar/utils/__init__.py

```diff
@@ -1,5 +1,5 @@
-import asyncio
 import json
+import logging as log
```

Comment:
I see this is a bug in the current implementation, but could we just fix the module use and make that `logging` instead of changing the name here? I'd prefer to not rename if not needed.

Reply:
sure, done!